spec/schema.json

```diff
@@ -623,10 +623,26 @@
         }
       },
       "BeneficialOwnerCorporate": {
+        "required": [
+          "name",
+          "principal_address",
+          "is_service_address_same_as_prinicpal_address",
+          "start_date",
+          "law_governed",
+          "legal_form",
+          "is_on_sanctions_list"
```

Comment:
Swagger spec will need changing to implement something similar to what I've just done in this PR, to reflect that at least one NOC must be ticked.

https://github.com/companieshouse/overseas-entities-api/pull/102.